internal/resources/resource_host.go

```diff
@@ -261,10 +286,24 @@ func HostResource() *schema.Resource {
 		},
 		Schema:      hostSchema(),
 		Description: "Provides Host resource. This allows Metal Host creation, deletion and update.",
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(sixty * time.Minute),
+			Update: schema.DefaultTimeout(sixty * time.Minute),
+			Delete: schema.DefaultTimeout(sixty * time.Minute),
+		},
 	}
 }
 
-//nolint: funlen    // Ignoring function length check on existing function
+func isHostActionAsync(d *schema.ResourceData) (bool, error) {
+	isAsync, ok := d.Get(hHostActionAsync).(bool)
+	if !ok {
+		return true, fmt.Errorf("%v is expected to be a bool", hHostActionAsync)
```

Comment:
Looks like where this is used, the first value is irrelevant if err != nil. Typically default value is returned if that's the case, e.g. false or nil or "".
```suggestion
		return false, fmt.Errorf("%v is expected to be a bool", hHostActionAsync)
```